src/get-started/java-binding.md

```diff
@@ -1,8 +1,72 @@
 # Java Binding User Guide
 
-**Java Binding App** is an application that includes the Exonum framework
+**Exonum Java App** is an application that includes the Exonum framework
 and Java services runtime environment.
 
+## Installation
+
+To run a node with your Java service you need to use Exonum Java application.
+You can download an archive containing the application and all the necessary
+dependencies on [the Releases page][github-releases] on GitHub. We suggest using
+`debug` version during development and `release` for production builds.
+Follow these steps:
+
+- Install [Maven 3][maven-install] which is essential for developing and building
+  Java service.
+- Install [Libsodium][libsodium-gihub] development library for your system.
+  You can use the following command for Ubuntu:
+
+  ```bash
+  sudo apt-get update && sudo apt-get install libsodium-dev
+  ```
+
+  For Mac OS, you can use [Homebrew][brew-install] as follows:
+
+  ```bash
+  brew install libsodium
+  ```
+
+- Download and unpack the archive from the [Releases page][github-releases]
+  into some known location. To install the latest release to `~/bin`:
+
+  ```bash
+  mkdir -p ~/bin
+  cd ~/bin
+  unzip /path/to/downloaded/exonum-java-0.6.0-release.zip
+  ```
+
+- Create an environment variable `EXONUM_HOME` pointing at this
+  location. You can also add an entry to the `PATH` variable.
```

Comment:
Adding the `EXONUM_HOME` path to the `$PATH` should be a requirement.

Reply:
Changed to `should`. Don't want to use `must`, because it's not actually a strict requirement needed for working application.